src/onnx/parse_constant.cpp

```diff
@@ -43,7 +43,7 @@ struct parse_constant : op_parser<parse_constant>
         // return empty literal
         if(v.get_shape().elements() == 0)
         {
-            return info.add_literal(literal{v.get_shape().type()});
+            return info.add_literal(literal{migraphx::shape{v.get_shape().type(), {1}, {0}}, {0}});
```

Comment:
Is this is using the deduced template literal constructor? I think it would be using the literal constructor that gets a shape type.